pyximport.install() before import api crash

```
In [1]: import pyximport
In [2]: pyximport.install()
Out[2]: (None, <pyximport.pyximport.PyxImporter at 0x10e3bdfd0>)

In [3]: import statsmodels.api as sm
/Users/matthiasbussonnier/.pyxbld/temp.macosx-10.7-x86_64-2.7/pyrex/statsmodels/tsa/kalmanf/kalman_loglike.c:321:10: fatal error: 'numpy/arrayobject.h' file not found
#include "numpy/arrayobject.h"
         ^
1 error generated.
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
...
```

Full trace [here](https://gist.github.com/Carreau/7545432). I can go around by pximport.install() after, but just want to record the bug. I should be mostly on recent stable but don't have time to try master now. Thanks.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,4 +1,4 @@
-global-include *.csv *.py *.txt *.pyx *.c
+global-include *.csv *.py *.txt
 #scikits*.*
 include MANIFEST.in
 
@@ -44,4 +44,4 @@ include statsmodels/stats/libqsturng/LICENSE.txt
 include statsmodels/regression/tests/results/leverage_influence_ols_nostars.txt
 
 
-global-exclude *~ *.swp  *.pyc *.bak
+global-exclude *~ *.swp  *.pyc *.bak *.pyx

